adapters/folder/readFolder.go

```diff
@@ -290,6 +324,16 @@ func (la *LocalAssetBrowser) parseDir(ctx context.Context, fsys fs.FS, dir strin
 
 			// Read metadata from the file only id needed (date range or take date from filename)
 			if la.requiresDateInformation {
+				// try to get date from icloud takeout meta
```

Comment:
creation date from icloud metadata takes presedence over exif information. was unsure if thats ok?

Reply:
That's what is done for google takeouts. If user have changed something on ICloud, it worth been set in Immich